go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/dylanmei/iso8601 v0.1.0
 	github.com/dyson/certman v0.2.1
-	github.com/giantswarm/apiextensions/v3 v3.32.0
+	github.com/giantswarm/apiextensions/v3 v3.32.1-0.20210908101333-093943a90ae0
```

Comment:
this looks like a branch

Reply:
it's master since we did not release apiextensions yet :)

Reply:
then what about releasing the apiextensions and update this here :) ?

Reply:
that's the plan. We also need to do that in the operator.